wikipedia-sentences/flows/index.yml

```diff
@@ -1,9 +1,9 @@
-!Flow
+!Flow                             # Define the flow used for indexing in this file
 version: '1'
-pods:
-  - name: encoder
-    uses: pods/encode.yml
+pods:                             # All pods that are executed in this flow are listed here
+  - name: encoder                 # Name for the encoder pod
```

Comment:
Assigned name right? The user can call this whatever they want. Perhaps we make that clear?